internal/tasks/fleet_rollout.go

```diff
@@ -148,6 +148,7 @@ func (f FleetRolloutsLogic) RolloutDevice(ctx context.Context) error {
 
 	if api.IsStatusConditionTrue(device.Status.Conditions, api.DeviceMultipleOwners) {
 		f.log.Warnf("Device has multiple owners, skipping rollout")
```

Comment:
Is this still a warn in this case? seems very unexpected

Reply:
done